kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/modem/quectel/bg96/QuectelBG96ModemFactory.java

```diff
@@ -0,0 +1,97 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2016 Eurotech and/or its affiliates
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.net.admin.modem.quectel.bg96;
+
+import java.util.Hashtable;
+
+import org.eclipse.kura.net.admin.NetworkConfigurationService;
+import org.eclipse.kura.net.admin.util.AbstractCellularModemFactory;
+import org.eclipse.kura.net.modem.CellularModem;
+import org.eclipse.kura.net.modem.ModemDevice;
+import org.eclipse.kura.net.modem.ModemTechnologyType;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.FrameworkUtil;
+import org.osgi.service.io.ConnectionFactory;
+import org.osgi.util.tracker.ServiceTracker;
+
+/**
+ * Defines Quectel BG96 Modem Factory
+ *
+ * @author ilya.binshtok
+ *
+ */
+public class QuectelBG96ModemFactory extends AbstractCellularModemFactory<QuectelBG96> {
+
+    // private static final Logger s_logger = LoggerFactory.getLogger(TelitHe910ModemFactory.class);
```

Comment:
If not needed remove commented lines. Please also adapt variable names

Reply:
ok done